_data/gaming.yml

```diff
@@ -314,47 +312,47 @@ websites:
     url: https://store.steampowered.com/
     img: steam.png
     tfa:
-      - email
-      - software
+      - proprietary
     doc: https://support.steampowered.com/kb_article.php?ref=8625-WRAH-9030
-    exception: "Software implementation requires use of the Steam Mobile app. Authentication on non-phone devices is not officially supported. SMS support is limited to receiving account recovery codes for password resets."
 
   - name: Tibia
     url: https://www.tibia.com/
     img: tibia.png
     tfa:
-      - software
+      - totp
     doc: https://www.tibia.com/support/?subtopic=gethelp&topicid=21
 
   - name: Toontown Rewritten
     url: https://www.toontownrewritten.com/
     img: toontownrewritten.png
     tfa:
-      - software
+      - totp
     doc: https://www.toontownrewritten.com/help/faq/accounts--login#twostep
 
   - name: Trion Worlds
     url: https://www.trionworlds.com/en/
     img: trionworlds.png
     tfa:
       - email
-      - software
+      - proprietary
     doc: https://support.trionworlds.com/hc/en-us/articles/204183357
 
   - name: Ubisoft/Uplay
     url: https://uplay.ubi.com/
     img: uplay.png
     tfa:
       - email
-      - software
+      - totp
+      - sms
     doc: https://support.ubi.com/en-GB/Faqs/000039953
 
   - name: Wargaming
     url: https://wargaming.net/
     tfa:
-      - software
+      - totp
+      - proprietary
```

Comment:
I don't see anything about a proprietary app?

Reply:
I was thrown off by this: https://ritastatusreport.live/2017/06/25/wargaming-authentication-app/

But it's using the same QR code, and adds, well, nothing :man_facepalming: